mobilitydb/src/point/tpoint.c

```diff
@@ -94,6 +94,7 @@ void
 temporalgeom_init()
 {
   lwgeom_set_handlers(palloc, repalloc, pfree, pg_error, pg_notice);
+  return;
```

Comment:
_Adding empty return to void functions_ could also have been a separate PR to simplify things.

Reply:
Noted for future PRs